### PR TITLE
e2e: add test for case where an empty secret is populated later

### PR DESCRIPTION
**Pull request by akshaysngupta**
## Checklist
- [x] The title of the PR is clear and informative
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] Issues addressed by the PR are mentioned in the description followed by `Fixes`.

## Description

This PR adds an E2E for a case where an empty secret is created without certificate information and is later populated with cert and key.
This simulates a case similar to Let's Encrypt where cert maybe empty in the beginning.

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: testsecret-tls
type: kubernetes.io/tls
data:
  tls.crt: ""
  tls.key: ""
```

Related to: https://github.com/Azure/application-gateway-kubernetes-ingress/pull/1070

### CODE REVIEW

File: scripts/e2e/cmd/runner/mfu_one_namespace_one_ingress_test.go [hunk 1]
```diff
@@ -359,6 +359,40 @@ var _ = Describe("MFU", func() {
...
+			Expect(err).To(BeNil())
+
+			EmptySecretYamlPath := "testdata/one-namespace-one-ingress/empty-secret/app.yaml"
+			klog.Info("Applying yaml: ", EmptySecretYamlPath)
+			err = applyYaml(clientset, namespaceName, EmptySecretYamlPath)
+			Expect(err).To(BeNil())
+			time.Sleep(30 * time.Second)
+
+			SecretYamlPath := "testdata/one-namespace-one-ingress/empty-secret/secret.yaml"
+			klog.Info("Applying yaml: ", SecretYamlPath)
```
Comment (3quanfeng) [inline scripts/e2e/cmd/runner/mfu_one_namespace_one_ingress_test.go:380]: Update secret yaml

**Review by 3quanfeng — APPROVED** [review]
